base_layer/wallet/migrations/2021-06-21-141831_remove_height_from_output/down.sql

```diff
@@ -0,0 +1,26 @@
+PRAGMA foreign_keys=off;
+ALTER TABLE outputs RENAME TO outputs_old;
```

Comment:
**NIT:** Just wondering if it is worth it to alter the old table and not just drop it?

Reply:
This is really the better way to do migrations I think. It copies any data from the old table into the new one and then drops the old table. The main reason we didn't do it this way in `2021-03-23-082938_update-outputs-for-tari-script` was because in that migration we were adding fields for which we couldn't create sensible default values in the Sql itself. 

Secondly the drop.sql is not ever actually going to be used but I still felt it worthwhile attempting to respect the migration contract.